contrib/ansible/openshift/register.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+default_tools_interval=$1
+ose_master_interval=$2
+ose_node_interval=$3
+file="/tmp/file"
+tool_dir=/var/lib/pbench-agent/tools-default
+
+pbench-stop-tools
+pbench-kill-tools
+pbench-clear-tools
```

Comment:
For each of the above calls, check that the stop/kill/clear tools are successful, return 0, before proceeding.